lib/verdict/experiment.rb

```diff
@@ -276,4 +285,11 @@ def is_scheduled?
     end
     return true
   end
+
+  def is_stop_new_assignments?
+    if @schedule_stop_new_assignment_timestamp and @schedule_stop_new_assignment_timestamp < Time.now
```

Comment:
You could simply return `@schedule_stop_new_assignment_timestamp and @schedule_stop_new_assignment_timestamp < Time.now`.